extensions/dicom-video/src/viewports/OHIFCornerstoneVideoViewport.js

```diff
@@ -1,51 +1,36 @@
-import React, { useCallback, useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import cornerstoneTools from 'cornerstone-tools';
-import cornerstone from 'cornerstone-core';
-import CornerstoneViewport from 'react-cornerstone-viewport';
-import OHIF, { DicomMetadataStore, utils } from '@ohif/core';
-import {
-  Notification,
-  ViewportActionBar,
-  useViewportGrid,
-  useViewportDialog,
-} from '@ohif/ui';
-import { adapters } from 'dcmjs';
-import id from '../id';
-
 
 function OHIFCornerstoneVideoViewport({
-  children,
-  dataSource,
   displaySet,
-  viewportIndex,
-  servicesManager,
-  extensionManager,
 }) {
-  const {
-    DisplaySetService,
-    MeasurementService,
-    ToolBarService,
-  } = servicesManager.services;
   const { videoUrl } = displaySet;
   const mimeType = "video/mp4";
```

Comment:
maybe move this outside of the function and make it all caps since it is constant

Reply:
Rather leave it where it is since it is supposed to be a computed field, just that the code to computed it isn't yet added.